Spencer puppetconf

sseebald opened the pull request:
Updated Puppetfile to include 2 modules: 
wsus  - forked from the forge module. Removed the management of the service wuauserv so it can be in the baseline profile
puppetconf_wsus - moved out the ps1 and the facter fact from the profile to a separate module

Updated baseline to have the service wuauserv management

Updated profile::windows::wsus to have a parameter for wsus server.

Commit:
Initial commit. Modified profile::windows::baseline, profile::windows::registry, profile::windows::wsus

Commit:
Update wsus.pp

Commit:
Update baseline.pp

Remove service management - Service is now managed in WSUS module

Commit:
Update registry.pp

Commit:
Update pending.rb

Commit:
Update baseline.pp

Commit:
Update baseline.pp

Commit:
Update baseline.pp

Commit:
Update baseline.pp

Commit:
Updated baseline. Removed the rename-admin PS exec

Commit:
Merge branch 'spencer_puppetconf' of https://github.com/puppetlabs/pmm-puppet-site into spencer_puppetconf

Commit:
Updated Puppetfile, wsus, and registry.

Commit:
re-adding in wuauserv management into baseline.pp

ccaum commented:
So is this good to merge? @glennsarti @Iristyle can you two weigh in?

glennsarti commented on site/profile/manifests/windows/baseline.pp:
```diff
@@ -8,6 +8,11 @@
 
+  service { 'wuauserv':
+    ensure => running,
+    enable => true,
```
Need to watch out for this.  On Server 2012 Windows Update is a Manual service with a Trigger.  Whereas, enable => true will set it to Automatic.

Happy to merge but a point for later e.g. PUP-6641

glennsarti commented:
May want to squash all the commits if you have time.

Commit:
Update Puppetfile

Moved from chocolatey/chocolatey to puppetlabs/chocolatey, added in chocolatey/chocolatey_server

Commit:
Update Puppetfile

sseebald committed:
Merge branch 'production' into spencer_puppetconf

Conflicts:
	Puppetfile
